### DISCUSSION TITLE
Channel Edit Philips 856

### DISCUSSION THREAD

**Discussion by veldenvl** [PredatH0r/ChanSort#325] [Q&A]
Hi! Is it possible to edit channels on Philips 856? Channel version - 115.0

**Comment by PredatH0r** [comment]
ChanSort has built-in support for the Philips format version 115.
There are a few caveats though. I think it is required that your channel search was done without any predefined network operators that cause a predefined order and automatic channel update should be disabled on your TV. Otherwise it might rearrange the channels again after the import.

**Reply by veldenvl** [reply]
I managed to upload a list of channels to ChanSort. After that I edited the channel list and got a text file ChanLst.txt . What should I do with it next?

**Comment by PredatH0r** [comment]
You probably exported a "reference list" by mistake. There is no ".txt" for normal Philips lists.
Philips lists don't consist of a single file, but instead of many files spread across a directory structure.
You typically export the list to USB on your TV, load the list into ChanSort, edit it, save it, and then import it back in the TV.
